include/tvm/relay/attrs/nn.h

```diff
@@ -1203,6 +1203,36 @@ struct SubPixelAttrs : public tvm::AttrsNode<SubPixelAttrs> {
   }
 };  // struct SubPixelAttrs
 
+/*! \brief Attributes used in correlation operators */
+struct CorrelationAttrs : public tvm::AttrsNode<CorrelationAttrs> {
+  int kernel_size;
+  int max_displacement;
+  int stride1;
+  int stride2;
+  Array<IndexExpr> padding;
```

Comment:
Should we change it to Array<Integer> ?

Reply:
What do you suggest? Isn't it `Array` already?

Reply:
Sorry somehow the github does not display `<>` correctly, updated

Reply:
I'm using the helper function `void GetPaddingHeightWidth(const Array<IndexExpr>& padding, IndexExpr* pad_h,
                                  IndexExpr* pad_w)'` which is also used by convolution. What about keeping `Array<IndexExpr>` so that we can migrate to `Array<Integer>` at once in the future?